src/common/session.js

```diff
@@ -0,0 +1,101 @@
+import { stringify } from 'querystring'
+import * as setCookie from 'set-cookie-parser'
+import { fetch } from './network'
+
+/**
+ * Session is used for auto storing cookies within all requests
+ * It also provide simple interfaces for major request types
+ */
+export class Session {
```

Comment:
There is no need to provide explicit cookie handling. They are managed implicitly. If it doesn't work with your bank please place session specific code in plugin directory.

Reply:
What do you mean that they are managed implicitly? I want to automatically store set cookies within requests, 'fetch' doesn't have such behavior
I've moved this class to plugin directory